src/rules/deprecation.ts

```diff
@@ -32,7 +37,7 @@ export default createRule<Options, MessageIds>({
     type: 'problem',
     docs: {
       description: 'Do not use deprecated APIs.',
-      recommended: 'warn',
+      recommended: 'recommended',
```

Comment:
What does this thing affect in docs btw?

Reply:
Actually, I think we may want to delete this line – I couldn't find that the previous way of setting recommended: 'warn' makes sense for custom rules in typescript-eslint either, and it doesn't match the fact that it's set to 'error' in this package's recommended config:
https://github.com/gund/eslint-plugin-deprecation/blob/df4dc2319d058ac0b367d71abe7585624b9510ce/src/configs/recommended.ts#L4

typescript-eslint themselves reworked their 'recommended' setup, into multiple new configs, which is why the types have changes, but the property does not seem to be documented.

Reply:
Hmm, okay then let's delete it altogether because it does not make any sense with `recommended` value anyway 😊